features/bootstrap/RequestContext.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\BeforeFeatureScope;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Artisan;
+use Illuminate\Support\Facades\DB;
+use KrzysztofRewak\Larahat\Helpers\DisablingThrottling;
+use PHPUnit\Framework\Assert;
+
+class RequestContext implements Context
+{
+    use DisablingThrottling;
+
+    private JsonResponse $response;
+    private $request;
+    private $nestedArray;
+
+    /**
+     * Initializes context.
+     *
+     * Every scenario gets its own context instance.
+     * You can also pass arbitrary arguments to the
+     * context constructor through behat.yml.
+     */
+    public function __construct()
+    {
+    }
+
+    /** @BeforeFeature */
+    public static function setupFeature(BeforeFeatureScope $scope): void
+    {
+        Artisan::call('migrate:refresh');
+    }
+
+    /**
+     * @Given :method is being sent to :endpoint
+     */
+    public function IsBeingSentTo(string $method, string $endpoint): void
+    {
+        $this->request = Request::create($endpoint, $method);
+        $this->request->headers->set('accept', 'application/json');
+    }
+
+    /**
+     * @When request is sent with debug
+     */
+    public function isSentWithDebug(): void
+    {
+        $this->response = app()->handle($this->request);
+        dd($this->request);
```

Comment:
It's better to use Xdebug than dumping and dying anywhere. You can ask @mtracz for help with configuration for Docker/PHPStorm.